web/srv/api_handlers.go

```diff
@@ -118,6 +124,20 @@ func (h *handler) handleAPIServices(w http.ResponseWriter, req *http.Request, p
 }
 
 func (h *handler) handleAPIStat(w http.ResponseWriter, req *http.Request, p httprouter.Params) {
+	// Initialize stat cache if needed
+	if h.statCache == nil {
+		// Cached items expire in 1.5s, expired ones are cleaned up every 5m
+		h.statCache = cache.New(1500*time.Millisecond, 5*time.Minute)
```

Comment:
Maybe these nums can be put in a constant or made configurable somehow so they dont stay magic ones